json-endpoints/jetpack/class.jetpack-json-api-get-option-backup-endpoint.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+class Jetpack_JSON_API_Get_Option_Backup_Endpoint extends Jetpack_JSON_API_Endpoint {
+	// /sites/%s/options/backup      -> $blog_id
+
+	protected $needed_capabilities = array(); // This endpoint is only accessible using a site token
+	protected $option_names;
+
+	function validate_input( $object ) {
+		$query_args = $this->query_args();		
+
+		if ( empty( $query_args['name'] ) ) {
+			return new WP_Error( 'option_name_not_specified', __( 'You must specify an option name', 'jetpack' ), 400 );
+		}
+
+		if ( is_array( $query_args['name'] ) ) {
+			$this->option_names = $query_args['name'];
```

Comment:
I think doing `$this->option_names = (array) $this->option_names` does the same trick